Engine/cs_par_dispatch.c

```diff
@@ -135,110 +129,7 @@ TREE *csp_locks_insert(CSOUND *csound, TREE *root)
 {
     csound->Message(csound,
                     Str("Inserting Parallelism Constructs into AST\n"));
-
     TREE *anchor = NULL;
-//
```

Comment:
This commented code was removed which was code that was in CS6 for parallel csound. Does this imply we will drop parallel csound? I'm alright if so, as it was never quite right, but want to confirm.

Reply:
it seemed to me to be an abandoned bit of code from a while ago. I am not intending to remove the parallel code. If you think we should leave that old commented out code there, we can revert this.

Reply:
I went back to csound6 and saw that this file isn't even in that branch anymore. I'm guessing it was there when the original work for cs7 started but was removed at some point. It is also not used in cmake at all so I think this whole file can be removed.

Reply:
I went ahead and removed the file and csound built and ran csdtests fine here. I've committed that removal.